Network Path - New Autonomous Systems view page (#37282)
DataDog/documentation: content/en/network_monitoring/network_path/as_view.md

---
title: Autonomous Systems View
description: Investigate the Network Path Autonomous Systems View

> is_beta: true

Reviewer: This param only applies to integrations and security rules according to our [internal docs](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1181582564/Frontmatter+parameters), so I think it's okay to remove!
```suggestion
```

further_reading:
- link: "/network_monitoring/network_path/list_view"
  tag: "Documentation"
  text: "Learn more about the List View in Network Path"
- link: "/network_monitoring/network_path/path_view"
  tag: "Documentation"
  text: "Learn more about the Path View in Network Path"
- link: "/network_monitoring/network_path/glossary"
  tag: "Documentation"
  text: "Network Path terms and concepts"
- link: "/network_monitoring/network_path/setup"
  tag: "Documentation"

>   text: "Setup Network Path"

Reviewer: ```suggestion
  text: "Set up Network Path"
```
One of those weird phrases that is one word as a noun and two as a verb!

Author: Facepalm...thank you for catching this!

---

## Overview

> The Autonomous Systems (AS) View provides visibility into the network providers and Internet Service Providers that carry your traffic across the BGP (Border Gateway Protocol) routing layer. This view monitors latency and performance metrics for every AS in your network paths, helping you pinpoint exactly which upstream providers are experiencing issues when your network performance degrades.

Reviewer: ```suggestion
The Autonomous Systems (AS) View provides visibility into the network providers and internet service providers (ISPs) that carry your traffic across the Border Gateway Protocol (BGP) routing layer. This view monitors latency and performance metrics for every AS in your network paths, helping you pinpoint exactly which upstream providers are experiencing issues when your network performance degrades.
```
Looks like we don't capitalize Internet Service Provider in other areas in the docs. Thought adding in the parenthetical would lead in nicely to other uses of ISP later on the page, even though it's a common acronym 🙂 

And then BGP looked a little weird in reverse order, so I changed that one too, but feel free to reverse it if it's also a really commonly understood acronym!

BGP routing issues and provider-specific problems are difficult to diagnose because they sit outside your direct control. The AS View makes these invisible layers visible, giving you the data to answer questions like "Is this a peering issue?" or "Did our traffic shift to a different transit provider?" without manually tracing routes or parsing BGP tables.

> Select [**Autonomous Systems (AS)**][1] in the [Network Path Explorer][2] to get started. 

Reviewer: ```suggestion
To get started, go to the Network Path Explorer and click [**Autonomous Systems (AS)**][1].
```
We do this sometimes in Security to reduce the number of links we give people, and thus try to cut down on the mental load of navigating a little. Happy to leave this one up to you!

Country
: The countries where traffic is observed for the AS.

Prefixes Monitored
: The IP prefixes observed for the AS across your monitored paths.

Tests Found
: The number of tests traversing the AS.

Issues Detected
: Issues observed for the AS, such as latency spikes or packet loss.

Use the filter controls above the list to narrow the results by **AS Number**, **Country**, **Category**, or **Issues Detected**.

## Autonomous System details

Click an Autonomous System in the list to open its details. The detail view includes a **Traffic** tab, a **Neighbors** tab, and a path list.

### Traffic

The **Traffic** tab shows a relational diagram of traffic flowing from **Upstream** sources through the selected AS to **Downstream** destinations. Hover over a traffic node to view its aggregate paths and number of occurrences, and click on any AS to filter to its paths in the [path list](#path-list).

### Neighbors

The **Neighbors** tab shows a full visualization of the upstream and downstream Autonomous Systems that neighbor the one you selected. Click on any AS in the graph to filter to its paths in the [path list](#path-list).

### Path list

> The path list includes individual paths through the AS, with the following columns:

Reviewer: Might be worth mentioning that you can click a path to open it in the Network Path Explorer?
